Changes

```diff
@@ -3,6 +3,12 @@ Working version
 
 (Changes that can break existing programs are marked with a "*")
 
+### Code generation and optimizations:
+
+- GPR#2278: Remove native code generation support for 32-bit Intel Darwin
```

Comment:
"Darwin" is a code name from the early 2000's.  Just call it "MacOS".

Reply:
I think Darwin should be mentioned as there have been non-Mac systems using that kernel.  I've updated the entry to mention macOS and iOS too.